tests/foreman/ui/test_errata.py

```diff
@@ -1102,27 +1102,36 @@ def test_positive_content_host_previous_env(
     :Setup:
         1. Make sure multiple environments are present.
         2. Content host's previous environments have additional errata.
+        3. Promote the Host's content view to a new lifecycle environment.
+        4. Set the Host to use the new environment, and original content view.
 
     :steps: Go to Content Hosts -> Select content host -> Errata Tab ->
-        Select Previous environments.
+        Select Previous environments (Environments Dropdown).
```

Comment:
Is it possible to perform this action on UI rather creating objects using api calls? Talking about [this](https://github.com/SatelliteQE/robottelo/pull/15548/files#diff-bd6863972772cca482646a04c83fba2589bdb45b1fcfdb0c4d544cd9fd17c920R1129-R1134) line 1129 - 1134

Reply:
I don't think so because this is a setup step and on 6.14 branch it is this way, I had mistakenly removed with the bz still open, could not fully test expected dropdown options

Reply:
Not sure if this case should also test, Host UI - Change content source/env via REX ?

Reply:
@vijaysawant Well this step, Selecting the dropdown option, is performed in UI, ln@1140, 1141 (after session)
I had been referencing Setup # 4, so think i may have misunderstood your question.

Reply:
yes, it is there.